en-US/snippets/2017/apr2017.lang

```diff
@@ -0,0 +1,41 @@
+## NOTE: Reference bug 1345221; Deadline: 21 March, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotion for Firefox Focus through snippet and email
+## NOTE: There is no character length limit
+
+
+# Campaign: Firefox desktop snippet
+;Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+
+
+# Campaign: Email subject line for Firefox Focus on iOS and Android
```

Comment:
You shouldn't put "Campaign" in all these comments, it makes sense only in the first one